website/client/js/services/achievementServices.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+/**
+ * Services that handle achievement logic.
+ */
+
+angular.module('habitrpg').factory('Achievement',
+['$rootScope', function($rootScope) {
+
+  function displayAchievement(achievementName) {
+    $rootScope.openModal('achievements/' + achievementName,
+      {controller:'UserCtrl', size:'sm'});
+  }
+
+  function displayBulkyAchievement(achievementName) {
+    $rootScope.openModal('achievements/' + achievementName,
+      {controller:'UserCtrl'});
```

Comment:
In that documentation, they list 'lg' as a possible value, not the default value.  Specifying the 'lg' parameter makes things look huge and weird:

![image](https://cloud.githubusercontent.com/assets/5641207/18374513/9f726a82-761b-11e6-99f4-4cafa88ff7be.png)